Dockerfile

```diff
@@ -34,165 +34,165 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-add-repository -y 'deb https://packages.erlang-solutions.com/ubuntu xenial contrib' && \
     apt-get -y update && \
     apt-get install -y --no-install-recommends \
-        advancecomp \
```

Comment:
Seems like this block until line 195 suffered from some unrelated indentation changes?

Reply:
yes, please ignore